### DISCUSSION TITLE
Some questions

### DISCUSSION THREAD

**Discussion by munerf** [privatenumber/instant-mocha#13] [Help wanted]
I'm trying to use instant-mocha with an existing config that I had, and it works, from mochapack.

The exact same webpack config file gives this error:

` - configuration has an unknown property 'default'. These properties are valid:`

Are there many setup differences between this project and the "old" one?

I'm using webpack 4.

**Comment by privatenumber** [comment]
This isn't a new version or fork of mochapack/mocha-webpack, but rather a new project that tackles the same goal.

Can you share your Webpack config or a minimal reproduction repository?

**Reply by munerf** [reply]
I don't think I can, or at least it would take a lot of time to do it. 
Is there a sample repo with webpack 4 that I can try to look at?

**Reply by privatenumber** [reply]
Looking at this again, it might be that your Webpack config was in ESM format (hence Webpack receiving `default` as a config property).

We added ESM support [here](https://github.com/privatenumber/instant-mocha/pull/28) (v1.4.0). Let me know if that fixes the issue.
